pkg/services/cloudmigration/cloudmigrationimpl/snapshot_mgmt.go

```diff
@@ -37,8 +37,21 @@ func (s *Service) getMigrationDataJSON(ctx context.Context, signedInUser *user.S
 		return nil, err
 	}
 
+	// Separate folders from dashboards
+	folderUids := make([]string, 0)
+	for i := 0; i < len(dashboards); {
```

Comment:
I find this kinda hard to read, could we move it to a function?

Reply:
Sure!

Reply:
[a9437ef](https://github.com/grafana/grafana/pull/90562/commits/a9437efc37f7d607c11f753dc4efb776cb4f7091)